oscarlab/graphene
# Reformat C sources, part 1

## Affected components


## Description of the changes

This is another part of #539. This PR reformats our codebase according to the clang-format rules we already have in the repo (which can be applied using `make format`) plus fixes some broken header dependencies we had in our repo.

To make this change as non-disruptive as possible, this PR touches only files which aren't modified by any currently open PR. I generated the list of files to modify using the following script:

```Python3
from github import Github
from pathlib import Path

g = Github('your_api_token___public_access_is_enough')
repo = g.get_repo('oscarlab/graphene')

prs = repo.get_pulls(state='open')
files_modified = []
for pr in prs:
	print(pr)
	files_modified += [f.filename for f in pr.get_files()]
print('--- Modified files: ---')
files_modified = list(set(files_modified))

all_files = []
p = Path('graphene-shallow-clone/')
for f in p.glob('**/*'):
	if f.parts[1] != '.git' and not f.is_dir():
		all_files.append(str('/'.join(f.parts[1:])))

for f in set(all_files) - set(files_modified):
	print('\'{}\''.format(f), end=' ')
print()
```

After some time I'll go through this whole process again and generate a next set of files which can be reformatted without any disruption to open PRs, until we reach the point where the whole codebase is correctly formatted.

I also plan to review all the changes by myself on Reviewable and possibly append some follow-up commits in case some places were incorrectly formatted.

## How to test this PR? 

Standard tests + check if the reformatting result is what we really desire (we may have bugs in our .clang-format config).


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/856)
